CHANGELOG.md

```diff
@@ -13,6 +13,8 @@
 * Allow standalone User Operator to modify status on KafkaUser
 * Add support for Kafka 2.4.1
 * Add support for Kafka 2.5.0
+* Remove ZooKeeper TLS sidecars and use native ZooKeeper TLS support instead
```

Comment:
```suggestion
* Remove TLS sidecars from ZooKeeper pods, using native ZooKeeper TLS support instead.
```

I think this makes it clearer that this change only affects the ZooKeeper server side of the connection, not the client side.